util/option_functions.R

```diff
@@ -0,0 +1,49 @@
+check_options <- function(opt, name_of_script_where_called) {
```

Comment:
To tie this into an earlier comment, I'm not sure if you need the `name_of_script_where_called` argument.